### DISCUSSION TITLE
Copilot for research

### DISCUSSION THREAD

**Discussion by shraddhabarke** [community/community#8391] [Copilot Conversations]
Hi everyone! I'm so excited to have gotten access to the preview version of GitHub Copilot! I'm a PhD student at UCSD, I had a couple of questions regarding the use of Copilot for a research study. In case of a remote study, can Copilot be shared and given to developers (who don't have access yet) for use? If not, can an exception be requested for research purposes?

**Comment by JuneNouh** [comment]
Well i did't get it yet... still waiting:)

**Comment by SmallLemur** [comment]
@shraddhabarke your access to Copilot is only for your account and cannot be shared. Your colleagues are welcome to apply to the Technical Preview though

**Reply by JuneNouh** [reply]
i got this email from you .. is that mean i got acsess to github copilot ?

**Reply by JuneNouh** [reply]
because i don't know why kind of email from Github to active my account ?

**Reply by SmallLemur** [reply]
@JuneNouh I was answering to @shraddhabarke, I cannot add you to Copilot. The team will progressively add people in the waiting list as more slots become available. Copilot being in Technical Preview, capacity on it is limited at the moment.

**Reply by JuneNouh** [reply]
OK thank you .. any idea about how long it takes ?

**Reply by SmallLemur** [reply]
I cannot give you a timeline. We are at capacity on the Technical Preview at the moment, we will resume onboarding as soon as we can. Thank you for your patience
